### PR TITLE
Code Sample for Authentication using jwt bearer in .net 7

### PR DESCRIPTION
**Pull request by khusroohayat**

### CODE REVIEW

**Comment by khusroohayat** [comment]
Hello @dodyg please review this PR

**Comment by dodyg** [comment]
@khusroohayat will do.

File: projects/.net7/authentication-4/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+
+namespace authentication_4.Controllers;
```
Comment (dodyg) [inline projects/.net7/authentication-4/Controllers/WeatherForecastController.cs:4]: remove the namespace

File: projects/.net7/authentication-4/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+namespace authentication_4;
```
Comment (dodyg) [inline projects/.net7/authentication-4/WeatherForecast.cs:1]: remove the namespace

**Review by dodyg — COMMENTED** [review]
Look nice. I just have two tiny comments.

**Pull request merged**